docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -142,9 +142,18 @@ return (
 ```
 
 :::warning
-This hook is not compatible with the deprecated column property `hide`, and you should not control the `columnVisibilityModel`.
+This hook is not compatible with the deprecated column property `hide`.
+It can be used with controlled `columnVisibilityModel` and `initialState`.
+To do so, provide the controlled value and the initial state to the hook as follow:
```

Comment:
You just have to provide one of the two, and the example below is redundant with what is currently existing